regolith/helpers/a_expensehelper.py

```diff
@@ -0,0 +1,179 @@
+"""
+Helper to add expenses.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "projecta"
+ALLOWED_TYPES = ["business", "travel"] # need to check all expense types.
+ALLOWED_STATI = ["submitted", "unsubmitted", "reimbursed"]
+
+
+def subparser(subpi):
+    subpi.add_argument("amount", help="expense amount",
+                       )
+    subpi.add_argument("name", help="A short name for the expense",
+                       default=None
+                       )
+    subpi.add_argument("purpose", help="A short description of expense purpose",
+                       default=None)
+    subpi.add_argument("reimbur")
+    subpi.add_argument("-b", "--business", action='store_true',
+                       help="expense type is business. If not specified defaults to travel"
+                       )
+    subpi.add_argument("-y", "--payee",
+                       help="payee of the expense, defaults to sbillinge"
+                       )
+    subpi.add_argument("-g", "--grants", nargs="+",
+                       help="grant, or list of grants that cover this expense. Defaults to tbd"
+                       )
+    subpi.add_argument("-s", "--status",
+                       help=f"status, from {ALLOWED_STATI}. Default is unsubmitted"
+                       )
+    subpi.add_argument("-z", "--segregated",
+                       help="Amount for any segregated expense. Defaults to 0"
+                       )
+    subpi.add_argument("-w", "--where",
+                       help="Where the expense has been submitted"
+                       )
+    subpi.add_argument("-n", "--notes", nargs="+",
+                       help="List of notes for the expense. Defaults to empty list"
+                       )
+    subpi.add_argument("-d", "--begin_date",
```

Comment:
I see the defaults mentioned in the help, but not any defaults set here.   Set them here if you can (check the argparse docs how to do it)

Reply:
I added a default, but I am not sure if should just leave it as a datetime object, or a string of today's date.

Reply:
date object, if it is allowed.

Reply:
done